.github/workflows/golang-ci-workflow.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
           # working-directory: somedir
 
           # Optional: golangci-lint command line arguments.
-          # args: --issues-exit-code=0
+          args: --timeout=5m --issues-exit-code=0
```

Comment:
oh maybe don't do `--issues-exit-code=0`?

Reply:
Fixed, accidentally put it from other tests. 👍